content/blog/2020-05-08-dvc-ambassador-program.md.md

```diff
@@ -0,0 +1,151 @@
+---
+title: 'Join the DVC Ambassador Program!'
+date: 2020-05-08
+description: |
+  We're launching our ambassador program for people all around the world to get involved in 
+  the DVC community.
+descriptionLong: |
+  We're launching our ambassador program for people all around the world to get involved in 
+  the DVC community.
+picture: 2020-05-08/Ambassador_Header.png
+author: marcel_rd
+commentsUrl: https://discuss.dvc.org/t/new-video-reimagining-devops-for-machine-learning/341
```

Comment:
fix it?

Reply:
yes will do right before we publish. don't want to make the comments post too early